HHS/Head-Start-TTADP
# [TTAHUB-788/816] remove trailing spaces

## Description of change

- Remove the (s) from the activity report tables
- Ignore spaces on either ends of text queries (not in the inputs, as we don't want to change what people are putting in there, just on the backend and in the url)

## How to test

- Verify the (s) has been removed
- Verify that you can put a space in a text query in the filter panel and it it doesn't effect the search results

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-788
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-816

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
